front/app/components/FormBuilder/components/FormFields/index.tsx

```diff
@@ -129,7 +111,6 @@ const FormFields = ({
                                 builderConfig={builderConfig}
                                 fieldNumbers={fieldNumbers}
                                 closeSettings={closeSettings}
-                                hasFullPageRestriction={isDragDisabled}
```

Comment:
Is this prop ever used anymore? If not, we should probably remove it from the component.